### ISSUE TITLE
Searchable content

### ISSUE THREAD

**Issue by DmitrySoshnikov** [esdiscuss/esdiscuss.org#42]
Search notes and topics.

**Comment by Jayflux** [comment]
I agree, maybe adding a google search box to the site would help!

**Comment by karlhorky** [comment]
Yeah a full-featured search (with separate page) would be a nice feature!

The current Algolia search box (seen below) is a good start but is limited and doesn't work on mobile:

![Screen Shot 2019-09-23 at 16 03 19](https://user-images.githubusercontent.com/1935696/65433161-cf4ea880-de1c-11e9-8b6c-a39030178d4e.png)

**Comment by ForbesLindesay** [comment]
I'd be very happy to see a custom search page implemented, with support for more advanced filtering by author etc.

**Comment by karlhorky** [comment]
Sounds good, I'll take a look tomorrow if I get some time!
